src/cmd.js

```diff
@@ -1,117 +1,197 @@
 #!/usr/bin/env node
 
-import minimist from 'minimist'
+
+import fs from 'fs'
```

Comment:
Using `require` here was intentional. We don't want to require in stuff we don't need (at least not here, since the commands themselves are mostly independent).
Should have added a comment. Sorry.